collector/cpu_linux.go

```diff
@@ -135,17 +139,18 @@ func NewCPUCollector(logger log.Logger) (Collector, error) {
 		logger:       logger,
 		isolatedCpus: isolcpus,
 		cpuStats:     make(map[int64]procfs.CPUStat),
+		config:       config,
 	}
-	err = c.compileIncludeFlags(flagsInclude, bugsInclude)
+	err = c.compileIncludeFlags(c.config.CPU.FlagsInclude, c.config.CPU.BugsInclude)
 	if err != nil {
 		return nil, fmt.Errorf("fail to compile --collector.cpu.info.flags-include and --collector.cpu.info.bugs-include, the values of them must be regular expressions: %w", err)
```

Comment:
Global statement about error messages referencing flags, might not need to change this for this initial push but since these can come from any source we should change to a more generic name.

Reply:
Yeah, i couldn't come with an elegant solution for now. We can change it later :+1: